--- siege-2.69/doc/Makefile.am~	2010-02-17 14:54:27.000000000 +0200
+++ siege-2.69/doc/Makefile.am	2010-02-17 14:55:55.178013240 +0200
@@ -47,11 +47,11 @@
 URLSTXT           =      $(sysconfdir)/urls.txt
 
 install-exec-hook: 
-	@if test -f $(SIEGERC); then \
-	  if cmp -s $(srcdir)/siegerc $(SIEGERC); then echo ""; \
+	@if test -f $(DESTDIR)$(SIEGERC); then \
+	  if cmp -s $(srcdir)/siegerc $(DESTDIR)$(SIEGERC); then echo ""; \
 	    else \
-	    echo ' $(INSTALL_DATA) $(srcdir)/siegerc $(SIEGERC).new'; \
-	    $(INSTALL_DATA) $(srcdir)/siegerc $(SIEGERC).new; \
+	    echo ' $(INSTALL_DATA) $(srcdir)/siegerc $(DESTDIR)$(SIEGERC).new'; \
+	    $(INSTALL_DATA) $(srcdir)/siegerc $(DESTDIR)$(SIEGERC).new; \
 	    echo "#####################################################"; \
 	    echo "WARNING: File $(SIEGERC) already exists."; \
 	    echo "         A new resource file has been installed as"; \
@@ -61,20 +61,20 @@
 	    echo "#####################################################"; \
 	  fi; \
 	else \
-	  $(INSTALL_DATA) $(srcdir)/siegerc $(SIEGERC); \
+	  $(INSTALL_DATA) $(srcdir)/siegerc $(DESTDIR)$(SIEGERC); \
 	fi 
-	@if test -f $(URLSTXT); then \
-	  if cmp -s $(srcdir)/siegerc $(URLSTXT); then echo ""; \
+	@if test -f $(DESTDIR)$(URLSTXT); then \
+	  if cmp -s $(srcdir)/siegerc $(DESTDIR)$(URLSTXT); then echo ""; \
 	    else \
-	    echo "WARNING: File $(URLSTXT) already exists."; \
+	    echo "WARNING: File $(DESTDIR)$(URLSTXT) already exists."; \
 	    echo "         It was NOT replaced with this installation."; \
 	  fi; \
 	else \
-	  $(mkinstalldirs) $(sysconfdir); \
-	  $(INSTALL_DATA) $(srcdir)/urls.txt $(URLSTXT); \
+	  $(mkinstalldirs) $(DESTDIR)$(sysconfdir); \
+	  $(INSTALL_DATA) $(srcdir)/urls.txt $(DESTDIR)$(URLSTXT); \
 	fi 
 
 
 uninstall:
-	rm -f $(SIEGERC)
+	rm -f $(DESTDIR)$(SIEGERC)
 
