diff -Nur siege-2.59.orig/src/Makefile.am siege-2.59/src/Makefile.am
--- siege-2.59.orig/src/Makefile.am	2004-01-06 21:14:35.000000000 +0100
+++ siege-2.59/src/Makefile.am	2004-03-24 19:52:59.449429232 +0100
@@ -28,9 +28,9 @@
 
 AM_CFLAGS          =   $(PTHREAD_CFLAGS) $(WARN_CFLAGS) $(SSL_CFLAGS)
 
-LDFLAGS            =   $(SSL_LDFLAGS) $(PTHREAD_LDFLAGS)  
+AM_LDFLAGS         =   $(SSL_LDFLAGS) $(PTHREAD_LDFLAGS)  
 
-LIBS               =   $(SSL_LIBS) 
+LIBS               =   $(SSL_LIBS)
 
 siege_LDADD        =   $(top_srcdir)/lib/joedog/libjoedog.la
 
